src/daemon/CMakeLists.txt

```diff
@@ -421,6 +421,48 @@ elseif (LINUX)
         endif ()
     endif ()
 
+    # A way to check for 66
+    if (EXISTS "/run/66" OR "66" IN_LIST FORCE_INIT_SYSTEM)
+        message(STATUS "66 detected")
+    if (EXISTS "/etc/66" )
+       set(66-PATH "/etc/66/service")
+    elseif (EXISTS "/var/lib/66/service")
+       set(66-PATH "/var/lib/66/service")
+    endif ()
+        set(CKB_NEXT_INIT_SYSTEM "66" CACHE INTERNAL "")
+        set(66_INSTALL_DIR "${66-PATH}" CACHE STRING "Where to install 66 service file.")
+
+        # Generate and import service
+        message(STATUS "Generating and importing ckb-next service (${CKB_NEXT_INIT_SYSTEM})")
+        configure_file(
+            "${ckb-next_SOURCE_DIR}/linux/66/ckb-next"
+            "${CMAKE_CURRENT_BINARY_DIR}/service/ckb-next"
+            @ONLY)
+
+        if (SAFE_INSTALL)
+            foreach (daemon IN ITEMS "ckb-daemon" "ckb-next-daemon")
```

Comment:
ckb-daemon no longer exists, so there's no chance someone can run it under 66. I don't see a point in keeping it here.